config/vufind/sitemap.ini

```diff
@@ -43,13 +43,37 @@ fileLocation   = /tmp
 ;    default will be "Solr,/Record/"
 index[] = "Solr,/Record/"
 ;index[] = "SolrAuth,/Authority/Record?id="
+;   Indexes can be completely disabled by setting this to false:
+;index = false
 
 ;  This setting controls how IDs are retrieved from the index. It may be
 ;  'search' (the default, most compatible but slower method), or 'terms' (the
 ;    faster option). Note that 'terms' method does not support hidden filters or
 ;    other limiting options and requires that the index has terms enabled.
 retrievalMode = search
 
+;  This settings controls whether different language versions are added to the
+;    sitemap. Possible values:
+;    empty or undefined - Language versions are not added
+;    "*"                - Language versions are added for all languages enabled in
+;                         config.ini
+;    "en,de,..."        - A comma-separated list of languages to include from the
+;                         ones enabled in config.ini
+;  Note that including language versions will make the sitemap files substantially
```

Comment:
The two notes here seem to have gotten deleted during your latest refactoring. I put them back because they still appeared to be relevant, but please feel free to revert if you had removed them intentionally.